Packs/Active_Directory_Query/Integrations/Active_Directory_Query/Active_Directory_Query.py

```diff
@@ -684,16 +724,28 @@ def search_group_members(default_base_dn, page_size):
     group_dn = args.get('group-dn')
     nested_search = '' if args.get('disable-nested-search') == 'true' else ':1.2.840.113556.1.4.1941:'
     time_limit = int(args.get('time_limit', 180))
+    account_name = args.get('sAMAccountName')
 
     custom_attributes: List[str] = []
-    default_attributes = DEFAULT_PERSON_ATTRIBUTES if member_type == 'person' else DEFAULT_COMPUTER_ATTRIBUTES
+
+    if member_type == 'person':
+        default_attributes = DEFAULT_PERSON_ATTRIBUTES
+    elif member_type == 'computer':
+        default_attributes = DEFAULT_COMPUTER_ATTRIBUTES
+    else:
+        default_attributes = DEFAULT_GROUP_ATTRIBUTES
```

Comment:
keeping the default as in the original code (should it be something else?).
```suggestion
    default_attribute_mapping = {
        'person': DEFAULT_PERSON_ATTRIBUTES,
        'group': DEFAULT_GROUP_ATTRIBUTES,
        'computer': DEFAULT_COMPUTER_ATTRIBUTES,
    }
    default_attributes = default_attribute_mapping.get(member_type, DEFAULT_COMPUTER_ATTRIBUTES)
```